src/pages/aurora-site/aurora-chat/index.jsx

```
@@ -1,119 +1,142 @@
-import { useState } from 'react';
-import ElizaBot from 'elizabot';
+import { useEffect, useRef, useState } from "react";
+import ElizaBot from "elizabot";
 import auroraImage from "@/assets/aurora.jpg";
```

ESLint

[error] 'auroraImage' is defined but never used.

(no-unused-vars)